pkg/providers/vsphere/vsphere.go

```diff
@@ -431,6 +435,30 @@ func (p *vsphereProvider) SetupAndValidateCreateCluster(ctx context.Context, clu
 		logger.Info("Skipping check for whether control plane ip is in use")
 	}
 
+	vuc := config.NewVsphereUserConfig()
+
+	if err := p.validator.validateUserPrivs(ctx, vSphereClusterSpec, vuc); err != nil {
+		return err
+	} else {
+		logger.MarkPass("EKSA_VSPHERE_USER vSphere privileges validated")
```

Comment:
All of these should probably reference "USERNAME" if that's how we documented and read them, and then maybe we can say the following instead. Or better if we can just output the value of the env var below instead
```suggestion
		logger.MarkPass("EKSA_VSPHERE_USERNAME user vSphere privileges validated")
```